Vurfan/Performance_Time.js

```diff
@@ -0,0 +1,7 @@
+function execution_Time() {
+  console.time("Total_Time_Taken");
```

Comment:
this is the non standard approach. do you know in production code, we are not allowed to use console??

Reply:
try something else